curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-functional-programming-by-building-a-spreadsheet/646d49bfff9079d4b38df115.md

```diff
@@ -135,7 +135,7 @@ const applyFunction = str => {
   const noHigh = highPrecedence(str);
   const infix = /([\d.]+)([+-])([\d.]+)/;
   const str2 = infixEval(noHigh, infix);
-  const functionCall = /([a-z]*)\(([0-9., ]*)\)(?!.*\()/i;
```

Comment:
@Nitz2611 I think you'll want to leave the regex unchanged except for the addition of the number set in the first capture group.

Reply:
done